src/test/cypress/fixtures/requests/programming_exercise_exam_template.json

```diff
@@ -0,0 +1,44 @@
+{
+    "mode": "INDIVIDUAL",
+    "includedInOverallScore": "INCLUDED_COMPLETELY",
+    "numberOfAssessmentsOfCorrectionRounds": [
+        {
+            "inTime": 0,
+            "late": 0
+        }
+    ],
+    "studentAssignedTeamIdComputed": false,
+    "secondCorrectionEnabled": false,
+    "type": "programming",
+    "bonusPoints": 0,
+    "isAtLeastTutor": false,
+    "isAtLeastEditor": false,
+    "isAtLeastInstructor": false,
+    "teamMode": false,
+    "assessmentDueDateError": false,
+    "dueDateError": false,
+    "presentationScoreEnabled": false,
+    "exerciseGroup": {},
```

Comment:
Why is this template necessary? we already have a programming_exercise_template

Reply:
Good catch. I think it was a relic, which is not needed anymore (It's not referenced in any code